### DISCUSSION TITLE
How to use Passport Strategy on a Resource Server to only introspect tokens?

### DISCUSSION THREAD

**Discussion by ghost** [panva/openid-client#605] [Q&A] [answered]
Hi, I have a resource server that needs to validate Oauth2 tokens without any authentication flow using passport. Is there any way to use your passport strategy to only introspect the access tokens without implementing any authentication flow? Or is this a use case not suited for this strategy?

**Comment by panva** [comment, marked as answer]
It's not a use case for openid-client.

**Reply by ghost** [reply]
Ok thank you
